### PR TITLE
Math Magicians: components #PR

### PR DESCRIPTION
**Pull request by Sonikak004**
### In this Branch I have: 👍 

- Deleted all the boilerplate from Create React App.
- Created a new directory components with calculator.js file.
- Implemented the design only (JSX and CSS).
- Used functional components.
- Used child component.
- Used calculator component in app.js file.
- Renders without problem in the browser.
- Fix linters.

### CODE REVIEW

File: src/components/rowButton.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <button className="button" type="button">2</button>
+        <button className="button" type="button">3</button>
+        <button className="button right-but" type="button">+</button>
+      </div>
+      <div className="row">
+        <button className="button bottom-button" type="button">0</button>
+        <button className="button" type="button">.</button>
+        <button className="button right-but" type="button">=</button>
+      </div>
+    </div>
```
Comment (cynthiainga) [inline src/components/rowButton.js:36]: - [x] As mentioned in the project requirements the component should match the given design, kindly make sure your design matches the original design (eg: colors, button size,...) 🙃 😉 
- [ ] And there's a comma next to your calculator that doesn't belong there 😁 

Your design

Screenshot 2023-07-19 at 13 48 30

Given design

Screenshot 2023-07-19 at 13 48 55

File: src/logo.svg
Comment (cynthiainga) [inline src/logo.svg:1]: - [x] As mentioned in the project requirements all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted, including the files and images in the `public` directory except the `index.html` file. Please kindly delete them 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Sonikak004, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [x] The project has a directory called `components` ✔️ 
- [x] There is a `Calculator.js` file inside the components directory ✔️ 
- [x] `Calculator.js` component is imported in `App.js` ✔️ 

## Required Changes ♻️

- [x] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Sonikak004 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- All linters checks passed ✔️ 
- Created a Calculator component ✔️ 
- Created a child component ✔️ 
- Calculator design is identical to that of the template ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x] The previous reviewer requested that you delete all boilerplates that were created with the create react act command. Kindly delete the following files:

- In the `public` directory, delete the following:
  - favicon.ico
  - manifest.json
  - robots.txt
- In the `src` directory, delete the following:
  - App.test.js

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Sonikak004 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Created a Calculator component ✔️ 
- Created a child component ✔️ 
- Calculator design is identical to that of the template ✔️ 
- All boilerplates were removed ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**